<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Théotime Grohens</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <link rel="stylesheet" href="style.css" />
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Théotime Grohens</h1>
</header>
<p><img src="./img/photo.jpg" width="250" align="right" style="margin:0px 20px"></p>
<p>I’m a Ph.D. student at <a href="https://www.insa-lyon.fr/">INSA
Lyon</a>, in the <a href="https://team.inria.fr/beagle/">Inria
Beagle</a> team.</p>
<p>Before that, I studied at the <a href="https://www.ens.psl.eu">École
Normale Supérieure</a> in Paris, where I obtained an M. Sc. in <a
href="https://wikimpri.dptinfo.ens-cachan.fr/doku.php">theoretical
computer science</a>, and an M. Sc. in <a
href="https://www.enseignement.biologie.ens.fr/spip.php?article193">ecology
and evolution</a>.</p>
<p>You can find my CV <a href="./pdf/resume.pdf">here</a>, follow me on
<a href="https://twitter.com/tgrohens">Twitter</a>, or look at my code
on <a href="https://github.com/tgrohens">GitHub</a> and on the <a
href="https://gitlab.inria.fr/tgrohens">Inria GitLab</a>.</p>
<p>You can reach me at <code>theotime (.) grohens (@) inria (.)
fr</code>.</p>
<h2 id="research-interests">Research interests</h2>
<p>My current research interests focus on evolutionary biology, and
especially mathematical and computational models of evolution.</p>
<p>The broad goal of my Ph.D. project is to gain a better understanding
of how epistatic interactions between different kinds of mutations help
shape evolutionary dynamics, and I am currently focusing on the coupling
between DNA supercoiling and gene transcription. In order to study this
phenomenon under the light of evolution, I develop and use <a
href="https://gitlab.inria.fr/tgrohens/evotsc">EvoTSC</a>, a simulation
aimed at understanding how the transcription-supercoiling coupling can
shape the genome over evolutionary timescales.</p>
<h2 id="publications">Publications</h2>
<h3 id="computational-biology">Computational Biology</h3>
<ul>
<li><p><strong><em>A Genome-Wide Evolutionary Simulation of the
Transcription-Supercoiling Coupling: extended version</em></strong>.
<strong>Théotime Grohens</strong>, Sam Meyer, Guillaume Beslon,
<em>Artificial Life (to appear)</em>. (<a
href="https://hal.archives-ouvertes.fr/hal-03667822/">hal</a>, <a
href="https://doi.org/10.1162/artl_a_00373">doi</a>, <a
href="./pdf/artl22.pdf">pdf</a>)</p></li>
<li><p><strong><em>A Genome-Wide Evolutionary Simulation of the
Transcription-Supercoiling Coupling</em></strong>. <strong>Théotime
Grohens</strong>, Sam Meyer, Guillaume Beslon, <em>ALIFE 2021</em>. (<a
href="https://hal.archives-ouvertes.fr/hal-03242696">hal</a>, <a
href="https://doi.org/10.1162/isal_a_00434">doi</a>, <a
href="./pdf/alife21.pdf">pdf</a>)</p></li>
</ul>
<h3 id="covid-19">Covid-19</h3>
<ul>
<li><strong><em>Understanding and monitoring the evolution of the
Covid-19 epidemic from medical emergency calls: the example of the Paris
area</em></strong>. Stéphane Gaubert, Marianne Akian, Xavier Allamigeon,
Marin Boyet, Baptiste Colin, <strong>Théotime Grohens</strong>, Laurent
Massoulié, David P. Parsons, Frédéric Adnet, Érick Chanzy, Laurent Goix,
Frédéric Lapostolle, Éric Lecarpentier, Christophe Leroy, Thomas Loeb,
Jean-Sébastien Marx, Caroline Télion, Laurent Tréluyer, Pierre Carli,
<em>Comptes Rendus Mathématique</em>, 2020. (<a
href="https://hal.archives-ouvertes.fr/hal-02648075v2">hal</a>, <a
href="https://doi.org/10.5802/crmath.99">doi</a>, <a
href="./pdf/crm20.pdf">pdf</a>)</li>
</ul>
<h3 id="programming-languages">Programming Languages</h3>
<ul>
<li><strong><em>Symbolic Execution for JavaScript</em></strong>. José
Fragoso Santos, Petar Maksimovic, <strong>Théotime Grohens</strong>,
Julian Dolby, Philippa Gardner, <em>Principles and Practice of
Declarative Programming</em>, 2018. (<a
href="https://doi.org/10.1145/3236950.3236956">doi</a>, <a
href="./pdf/ppdp18.pdf">pdf</a>)</li>
</ul>
<h2 id="talks-posters">Talks &amp; Posters</h2>
<ul>
<li>September 2022. <a
href="http://www.biosyl.org/news/biosyl-day">BioSyL</a>, Lyon, short
talk.</li>
<li>November 2021. <a
href="https://www.gdr-bim.cnrs.fr/journees-du-gdr-bim-2021/">GDR
BIM</a>, Lyon, poster.</li>
<li>November 2021. <a
href="https://project.inria.fr/compsysbio2021/">CompSysBio winter
school</a>, Aussois, short talk and poster.</li>
<li>July 2021. <a href="https://www.robot100.cz/alife2021/">ALIFE
2021</a>, Prague (virtual), short talk.</li>
<li>November 2019. <a
href="https://aiem2019.sciencesconf.org/program">GDR AIEM</a>, Toulouse,
short talk.</li>
</ul>
<h2 id="teaching">Teaching</h2>
<p>I teach Computer Science at <a href="https://www.insa-lyon.fr/">INSA
Lyon</a>, in the <a
href="https://biosciences.insa-lyon.fr/">Biosciences</a> and <a
href="https://if.insa-lyon.fr/">Computer Science</a> departments.</p>
<ul>
<li><p>In 2021-2022, I taught in the <a
href="https://if.insa-lyon.fr/fr/content/formation">Computer Science</a>
track:</p>
<ul>
<li>Introduction to C++ (TA, 16h)</li>
<li>Introduction to Parallel Programming (TA, 22h)</li>
</ul></li>
<li><p>In 2020-2021, I taught in the <a
href="https://biosciences.insa-lyon.fr/fr/content/parcours-bioinformatique-et-modelisation">Bioinformatics
and Modeling</a> track:</p>
<ul>
<li>Introduction to Theoretical Computer Science (12h)</li>
</ul>
<p>and in the <a
href="https://if.insa-lyon.fr/fr/content/formation">Computer Science</a>
track:</p>
<ul>
<li>Introduction to C++ (TA, 16h)</li>
<li>Introduction to Parallel Programming (TA, 22h)</li>
<li>Tutoring for last-year student projects (10h)</li>
</ul></li>
<li><p>In 2019-2020, I taught in the <a
href="https://biosciences.insa-lyon.fr/fr/content/parcours-bioinformatique-et-modelisation">Bioinformatics
and Modeling</a> track:</p>
<ul>
<li>Algorithms for Bioinformatics (10h)</li>
<li>Introduction to Theoretical Computer Science (12h)</li>
</ul></li>
</ul>
</body>
</html>
