---
title: Théotime Grohens
---

<img src="./img/photo.jpg" width="250" align="right" style="margin:0px 20px">

I’m a post-doctoral researcher at the [Center for Genomic Regulation](https://www.crg.eu/) in Barcelona, in the [Weghorn lab](https://weghornlab.net/).

Before that, I did my Ph.D. at [INSA Lyon](https://www.insa-lyon.fr/), in the [Inria Beagle](https://team.inria.fr/beagle/) team.
Even before, I studied at the [École Normale Supérieure](https://www.ens.psl.eu) in Paris, where I obtained an M. Sc. in [theoretical computer science](https://wikimpri.dptinfo.ens-cachan.fr/doku.php), and an M. Sc. in [ecology and evolution](https://www.enseignement.biologie.ens.fr/spip.php?article193).

You can find my CV [here](./pdf/resume.pdf), follow me on [Twitter](https://twitter.com/tgrohens), or look at my code on [GitHub](https://github.com/tgrohens) and on the [Inria GitLab](https://gitlab.inria.fr/tgrohens).

You can reach me at `theotime (.) grohens (@) crg (.) eu`.

## Research interests

My research interests focus on evolutionary biology, and especially mathematical and computational models of evolution.

I am currently working on understanding the selection pressures that are at work in cancer cells, and in particular on the interaction between cancer and the immune system.

During my Ph.D., I worked on understanding how epistatic interactions between different kinds of mutations help shape evolutionary dynamics, through the lens of the coupling between DNA supercoiling and gene transcription.
In order to study this phenomenon under the light of evolution, I developed and used [EvoTSC](https://gitlab.inria.fr/tgrohens/evotsc), a simulation aimed at understanding how the transcription-supercoiling coupling can shape bacterial genomes over evolutionary timescales.

## Publications

### Computational Biology

* **_Forward-in-time simulation of chromosomal rearrangements: The invisible backbone that sustains long-term adaptation_**.\
  Paul Banse, Juliette Luiselli, David P. Parsons, **Théotime Grohens**, Marco Foley, Leonardo Trujillo, Jonathan Rouzaud-Cornabas, Carole Knibbe, Guillaume Beslon. _Molecular Ecology_, 2023. ([doi](https://doi.org/10.1111/mec.17234), [pdf](./pdf/molevol2023.pdf))

* **_A Genome-Wide Evolutionary Simulation of the Transcription-Supercoiling Coupling: extended version_**.\
  **Théotime Grohens**, Sam Meyer, Guillaume Beslon, _Artificial Life_, 2022. ([hal](https://hal.archives-ouvertes.fr/hal-03667822/), [doi](
https://doi.org/10.1162/artl_a_00373), [pdf](./pdf/artl22.pdf))

* **_A Genome-Wide Evolutionary Simulation of the Transcription-Supercoiling Coupling_**.\
  **Théotime Grohens**, Sam Meyer, Guillaume Beslon, _ALIFE 2021_, 2021. ([hal](https://hal.archives-ouvertes.fr/hal-03242696), [doi](https://doi.org/10.1162/isal_a_00434), [pdf](./pdf/alife21.pdf))

#### Preprint(s)

* **_Emergence of Supercoiling-Mediated Regulatory Networks through Bacterial Chromosome Rearrangements_**.\
  **Théotime Grohens**, Sam Meyer, Guillaume Beslon, _bioRxiv_, 2022. ([doi](https://doi.org/10.1101/2022.09.23.509185), [pdf](./pdf/biorXiv22.pdf))

### Covid-19

* **_Understanding and monitoring the evolution of the Covid-19 epidemic from medical emergency calls: the example of the Paris area_**.\
  Stéphane Gaubert, Marianne Akian, Xavier Allamigeon, Marin Boyet, Baptiste Colin, **Théotime Grohens**, Laurent Massoulié, David P. Parsons, Frédéric Adnet, Érick Chanzy, Laurent Goix, Frédéric Lapostolle, Éric Lecarpentier, Christophe Leroy, Thomas Loeb, Jean-Sébastien Marx, Caroline Télion, Laurent Tréluyer, Pierre Carli, _Comptes Rendus Mathématique_, 2020. ([hal](https://hal.archives-ouvertes.fr/hal-02648075v2), [doi](https://doi.org/10.5802/crmath.99), [pdf](./pdf/crm20.pdf))

### Programming Languages

* **_Symbolic Execution for JavaScript_**.\
  José Fragoso Santos, Petar Maksimovic, **Théotime Grohens**, Julian Dolby, Philippa Gardner, _Principles and Practice of Declarative Programming_, 2018. ([doi](https://doi.org/10.1145/3236950.3236956), [pdf](./pdf/ppdp18.pdf))

### PhD Thesis

* **_Ride the Supercoiling: Evolution of Supercoiling-Mediated Gene Regulatory Networks through Genomic Inversions_**.\
  **Théotime Grohens**, 2022. ([pdf](./pdf/phd.pdf))


## Talks & Posters

* September 2022. [BioSyL](http://www.biosyl.org/news/biosyl-day), Lyon, short talk.
* November 2021. [GDR BIM](https://www.gdr-bim.cnrs.fr/journees-du-gdr-bim-2021/), Lyon, poster.
* November 2021. [CompSysBio winter school](https://project.inria.fr/compsysbio2021/), Aussois, short talk and poster.
* July 2021. [ALIFE 2021](https://www.robot100.cz/alife2021/), Prague (virtual), short talk.
* November 2019. [GDR AIEM](https://aiem2019.sciencesconf.org/program), Toulouse, short talk.

## Teaching

During my Ph.D., I taught computer science at [INSA Lyon](https://www.insa-lyon.fr/), in the [Biosciences](https://biosciences.insa-lyon.fr/) and [Computer Science](https://if.insa-lyon.fr/) departments, and at [IUT Lyon 1](https://iut.univ-lyon1.fr/).
